integration-libs/epd-visualization/components/visual-picking/visual-picking-tab/product-list/visual-picking-product-list.service.ts

```diff
@@ -57,6 +61,11 @@ export class VisualPickingProductListService {
     this.selectedProductCodes = [];
   }
 
+  ngOnDestroy(): void {
+    this.getFilteredProductReferences$Subscription.unsubscribe();
+    this.filteredItems$Subscription.unsubscribe();
```

Comment:
nitpick: `$` suffix is used by convention to highlight RxJs observables. It looks a bit funky in the middle of the property name. Let's remove it.
btw. `$` is also not appropriate as a sufix of methods names.

Reply:
I have renamed these items as requested.